src/lib/serviceRoutines/postSubscribeContext.cpp

```diff
@@ -51,6 +52,7 @@
 *
 * URI parameters
 *   - notifyFormat=XXX    (used by mongoBackend)
+*   - q
```

Comment:
"q" as URL param is only valid in "GET /v2/entities" operation (please check http://telefonicaid.github.io/fiware-orion/api/v2/)